### DISCUSSION TITLE
after "pip install apache-airflow-providers-microsoft-psrp" does not show up at "providers" section in Airflow 2.9.3

### DISCUSSION THREAD

**Discussion by foeyonghai** [apache/airflow#41713] [Q&A]
Hi all,

I am trying to get the microsoft-psrp provider available in Airflow. As you can see below it seems to be installed in the Docker container but does not show up. To be sure I rebooted the whole Ubuntu server but as expected that does not solve the thing.

![Capture2](https://github.com/user-attachments/assets/75ea7a64-7b6b-4953-a9c5-8da3a1547200)
Used airflow.sh to get into the container and switched to the "airflow" user.

![Capture3](https://github.com/user-attachments/assets/792c39c0-492f-426a-8b83-388b67432b60)
It seems to be installed successfully but...

![Capture4](https://github.com/user-attachments/assets/6550e3c2-4954-42a2-b309-14c3b33b071b)
What am I doing wrong? I don't get it at this point.

Many thanks!

**Comment by potiuk** [comment]
Look at warnings in the log of airflow when starting - likely you will see some import errors as likely the Python libraries of Microft to be installed in the system.
